py3dbp/auxiliary_methods.py

```diff
@@ -33,3 +35,47 @@ def set_to_decimal(value, number_of_decimals):
     number_of_decimals = get_limit_number_of_decimals(number_of_decimals)
 
     return Decimal(value).quantize(number_of_decimals)
+
+def _plotCube(bin, ax, x, y, z, dx, dy, dz, color='red', lw=4):
+    """ Auxiliary function to plot a cube. code taken somewhere from the web.  """
+    xx = [x, x, x+dx, x+dx, x]
+    yy = [y, y+dy, y+dy, y, y]
+    kwargs = {'alpha': 1, 'color': color, 'lw': lw}
+    ax.plot3D(xx, yy, [z]*5, **kwargs)
+    ax.plot3D(xx, yy, [z+dz]*5, **kwargs)
+    ax.plot3D([x, x], [y, y], [z, z+dz], **kwargs)
+    ax.plot3D([x, x], [y+dy, y+dy], [z, z+dz], **kwargs)
+    ax.plot3D([x+dx, x+dx], [y+dy, y+dy], [z, z+dz], **kwargs)
+    ax.plot3D([x+dx, x+dx], [y, y], [z, z+dz], **kwargs)
+
+def plotBoxAndItems(bin, export_img, title=""):
```

Comment:
Changes looks good ! However, name functions with with Python convention: `plot_box_and_items`.
I think pyCharm is giving hints for it. Look at them !

![image](https://user-images.githubusercontent.com/22714418/192966804-e06fcfe9-0ede-45bc-ba11-438b9260a2f2.png)

Reply:
OK